ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -173,22 +173,63 @@ descriptor:
   winter: "Method or process used during colder weather (i.e. winter season)."
   zone: "Region of building which is conditioned."
 measurement:
-  concentration: "Concentration of chemical (usually in parts per million or parts per billion)."
+  acceleration: "The rate of change of velocity per unit of time."
+  angle: "The space between two intersecting lines or surfaces at or close to the point where they meet."
+  angularmomentum: "The quantity of rotation of a body, which is the product of its moment of inertia and its angular velocity."
+  area: "The extent or measurement of a surface."
+  concentration: "The relative amount of a particular substance contained within a solution or mixture or in a particular volume of space."
   current: "Flow of electric charge."
+  density: "The quantity of mass per unit volume of a substance."
+  electriccapacitance: "The ratio of the change in an electric charge in a system to the corresponding change in its electric potential."
+  electricconductance: "The degree to which an object conducts electricity, calculated as the ratio of the current which flows to the potential difference present"
+  electricconductivity: "The degree to which a specified material conducts electricity, calculated as the ratio of the current density in the material to the electric field which causes the flow of current."
+  electriccurrentdensity: "The amount of electric current flowing per unit cross-sectional area of a material."
+  electricfield: "A region around a charged particle or object within which a force would be exerted on other charged particles or objects."
+  electricinductance: "The property of an electric conductor or circuit that causes an electromotive force to be generated by a change in the current flowing."
+  electricphasedifference: "The time interval by which a wave leads by or lags by another wave."
+  electromagneticfieldstrength: "Quantitative expression of the intensity of an electric field at a particular location"
```

Comment:
is this an electric field or a magnetic field? they measure different things; electric field is in V/m and magnetic field is in A/m.

Reply:
updated to electric